app/controllers/employment/EmployerInformationController.scala

```diff
@@ -45,8 +45,10 @@ class EmployerInformationController @Inject()(implicit val cc: MessagesControlle
 
       source match {
         case Some(EmploymentSourceOrigin(source, _)) =>
-          val (name, benefitsIsDefined) = (source.employerName, source.employmentBenefits.isDefined)
-          Ok(employerInformationView(name, employmentId, benefitsIsDefined, taxYear, isInYear))
+          val studentLoansCheck: Boolean = source.employmentData.flatMap(_.deductions).flatMap(_.studentLoans).isDefined
```

Comment:
I would add checking the amount fields aren't empty into here like
`uglDeductionAmount.isDefined || pglDeductionAmount.isDefined`

Might not be a problem depending on how we handle the model but think its more robust to check if there's any values defined